conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+* To create a successful collaboration across teams (UXD, CCS, PM, etc.) for every quick start
+* To define which teams are involved at each step of the quick start effort so that the end result is organized, clean, and in final form for users
+* To respect the bandwidth and input of each team
+
+#### Phase 1: Planning
+1. Determine which product area needs a quick start.
+  * Identify stumbling blocks or complex areas where we can enable user success with as little friction as possible. This information will be gathered from:
+    * UXD user research
+    * Pain points shared from internal feedback sessions (Solution Architects, Technical Marketing, CEE/Customer Support, etc.)
+    * Product Management input
+    * OpenShift or Kubernetes documentation
+    * Features we want to highlight or push more to entice users to try
+    * Areas we want to offer more opinionated views or flows. For example, there are likely cases where there are multiple ways for users to accomplish a task, but we’re recommending a method or suggesting how to get up and running with XYZ.
+  * Note that regardless of where the quick start idea originates, quick starts should always be a part of an epic.
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
+  * **Time frame:** Before epic prioritization
+  * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start. Epics are created in the relevant development project.
+2. Plan user flows and interaction design. (Optional)
+  * UXD Designer, UXD Content Strategist, and CCS Content Strategist and/or Writer meet with Product Manager to determine the quick start’s goal and where we need to reduce friction or confusion in completing some tasks.
+  * Collaborate on what the ideal user flow would look like. For example, what are the tasks and the sub tasks that users need to do in order to complete the quick start?
+  * Work with Abi to figure out who we work with in CCS and tag in JIRA story.
+    * Consider prioritization (track in JIRA); use template.
+    * Maybe need a separate process for external teams; have UXD Content & CCS just do a quick run-through.
+  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs.
+  * **Team responsible:** UXD Design (including Dev Tools UXD); Product Management; UXD Content Strategy; CCS
+  * **Time frame:** Conceptual design story (Sprint 1)
+  * **Definition of done:**
+    * UXD Designer creates interaction designs (not hi-fi) in Sketch and Marvel.
+    * Product Manager reviews the interactions and provides feedback.
+    * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 2: First draft
+The contributing team writes the first draft of the quick start, using the following as references:
```

Comment:
We originally had the designer listed as doing the first draft. Who on the contributing team will be doing the draft? We might want to specify that here so that we avoid ambiguity.

Reply:
I think this may vary from team to team, so maybe leaving it vague is our best bet here. @beanh66 thoughts?

Reply:
Agreed, could come from any team in RH

Reply:
Thanks, @beanh66 / @bmignano  ! So who is this document intended to be for? Is it supposed to be geared towards all RH teams doing quick starts, or is it for UXD doing quick starts? 

Right now, it's written in a way that's geared towards the UXD team, with information regarding working with product teams, PMs, QE, CCS, etc. I don't think that would apply to all teams at Red Hat (e.g., Marketing). If this is a resource for UXDers creating quick starts, then we should specify who's doing the writing. But if it's a resource for any Red Hat team creating quick starts, then we may need to revise all of it to be more general (and it should probably live somewhere that isn't OpenShift-specific, like maybe on The Source).